EventHubs/requirements.txt

```diff
@@ -1,7 +1,8 @@
-requests==2.20.0
-azure-mgmt-resource==1.2.2
-azure-mgmt-eventhub==1.2.0
+requests==2.31.0
+azure-monitor-query==1.2.0
+azure-identity==1.15.0
+azure-core==1.29.6
+azure-mgmt-resource==23.0.1
+azure-mgmt-eventhub==11.0.0
 azure-servicebus==0.21.1
```

Comment:
azure-servicebus - Should this be 7.11.14. Current version is around 7 years old

Reply:
azure-servicebus was used earlier in both metric and blob,
Now only blob files using 'azure-servicebus'.

and in this PR we only updated/modified the methods/libs for metric files...

Reply:
that's incorrect it's used in insert_mock_metrics function lets upgrade this library as well  https://github.com/SumoLogic/sumologic-azure-function/blob/fcc8a29faf650e04167ff572e0a74ea90cf269c3/EventHubs/tests/baseeventhubtest.py#L82